iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -21,38 +21,41 @@
 import android.support.annotation.VisibleForTesting;
 import android.util.Log;
 
+import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
```

Comment:
any reason to move org. block here?

Reply:
Android Studio formatted it as such; 'foreign' packages first (i.e. `android.*`, `org.eclipse.*`), then `java.*`, then `javax.*`. I can change it back but then it'll jump back to this location on each reformat.

Reply:
I understand. Let's keep your change.